squash/dashboard/urls.py

```diff
@@ -8,17 +8,19 @@
 api_router = DefaultRouter()
 api_router.register(r'jobs', views.JobViewSet)
 api_router.register(r'metrics', views.MetricViewSet)
-api_router.register(r'measurements', views.MeasurementViewSet,
-                    base_name='measurements')
 api_router.register(r'datasets', views.DatasetViewSet,
                     base_name='datasets')
+
+# endpoints for data consumed by the bokeh apps
+api_router.register(r'measurements', views.MeasurementViewSet,
+                    base_name='measurements')
 api_router.register(r'AMx', views.AMxViewSet, base_name='AMx')
 api_router.register(r'PAx', views.PAxViewSet, base_name='PAx')
 
 urlpatterns = [
+    url(r'^$', views.home, name='home'),
     url(r'^dashboard/api/', include(api_router.urls)),
     url(r'^dashboard/admin/', include(admin.site.urls)),
-    url(r'^dashboard/(?P<bokeh_app>[\w./-]+)/?$',
-        views.embed_bokeh, name='bokeh_app'),
-    url(r'$', views.home, name='home')
+    url(r'^dashboard/(?P<bokeh_app>[\w./-]+)/$', views.embed_bokeh,
+        name='embed_bokeh')
 ]
```

Comment:
<img width="657" alt="screen shot 2017-03-28 at 4 05 53 pm" src="https://cloud.githubusercontent.com/assets/13968086/24431402/89c29a4c-13d0-11e7-864e-41d700715bce.png">
404.html must be placed in the top level of the template tree.